py-polars/polars/expr/expr.py

```diff
@@ -3998,11 +4001,6 @@ def map_batches(
 
         Notes
         -----
-        If you are looking to map a function over a window function or group_by context,
-        refer to func:`map_elements` instead.
-
-        Warnings
-        --------
```

Comment:
same as above, the "Warnings" section is meant to be used to document the warnings which a function may throw. this looks more suited to a "Notes" section